modules/tinymce/src/core/test/ts/browser/html/StylesTest.ts

```diff
@@ -264,4 +271,16 @@ describe('browser.tinymce.core.html.StylesTest', () => {
     assertStyles(styles, 'color: rgba(1, 2, 3);', 'color: rgba(1, 2, 3);');
     assertStyles(styles, 'color: rgb(1, 2, 3, 0.5);', 'color: rgb(1, 2, 3, 0.5);');
   });
+
+  it('TINY-10916: transparent should not be converted to other format', () => {
+    const styles = Styles();
+    assertStyles(styles, 'color: transparent;', 'color: transparent;');
```

Comment:
Should add more test cases to cover other properties that accept color values such as `border-color` and `background-color`